File: RelacjeSportowe.DataAccess/Models/Transmission.cs
using RelacjeSportowe.DataAccess.Interfaces;
using System;
using System.Collections.Generic;

namespace RelacjeSportowe.DataAccess.Models
{
    public class Transmission : IEntity
    {
        public int Id { get; set; }

        public DateTime StartDate { get; set; }

        public DateTime? EndDate { get; set; }

        public virtual User User { get; set; }

        public int UserId { get; set; }

        public string TeamHome { get; set; }

        public string TeamAway { get; set; }

        public DateTime EventDate { get; set; }

        public virtual IEnumerable<TransmissionEvent> TransmissionEvents { get; set; }
    }
}

File: RelacjeSportowe.Web/ClientApp/src/app/models/transmission.ts
export class Transmission {
  public id: number;
  public endDate: Date;
  public eventDate: Date;
  public startDate: Date;
  public teamHome: string;
  public teamAway: string;
  public username: string;
  public viewersCount: string;

  constructor(data: any) {
    this.endDate = data.endDate;
    this.eventDate = data.eventDate;
    this.startDate = data.startDate;
    this.teamHome = data.teamHome;
    this.teamAway = data.teamAway;
    this.username = data.username;
    this.viewersCount = data.viewersCount;
  }
}